src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.TrackedGeneratorDriver.cs

```diff
@@ -28,7 +28,7 @@ internal TrackedGeneratorDriver(GeneratorDriver generatorDriver, bool needsFullG
             public GeneratorDriver? GeneratorDriver { get; }
             public bool NeedsFullGeneration { get; }
 
-            /* PROTOTYPE: re-enable when PendingEdit is public again
+            /* TODO: re-enable when PendingEdit is public again
```

Comment:
📝 This should use `#if false` instead of a block comment.

Reply:
Thanks, updated in the PR.